presto-native-execution/presto_cpp/main/QueryContextManager.cpp

```diff
@@ -51,13 +51,23 @@ folly::IOThreadPoolExecutor* spillExecutorPtr() {
 }
 
 namespace {
+std::string maybeRemoveNativePrefix(const std::string& name) {
+  static const std::string kNativePrefix = "native_";
+  const auto result =
```

Comment:
Will it work with old coordinator where native prefix is not passed? We can keep the worker backward compatible so that it can work with native and without native prefixed properties. Once coordinator is updated, we can remove anything that does not have native prefix.

Reply:
I think it will work if old coordinator doesn't pass native prefix.

Reply:
Nice. Then lets just enhance the unit test to keep the legacy case. Once coordinator is updated, we can remove it.

Reply:
Yes, it will work with old coordinator